aiven/client/cli.py

```diff
@@ -69,6 +69,8 @@ def convert_str_to_value(schema, str_value):
                                  .format(str_value, ", ".join(values)))
     elif "array" in schema["type"]:
         return [convert_str_to_value(schema["items"], val) for val in str_value.split(",")]
+    elif "null" in schema["type"]:
```

Comment:
Could also check that str_value is `None`, otherwise if we add new types later and forget to update this then the unsupported type gets converted into `None`, which is much harder to spot that getting explicit error.